src/com/rcv/ResultsWriter.java

```diff
@@ -571,4 +573,10 @@ public int compare(
     }
     return sortedCandidates;
   }
+
+  private enum OutputType {
```

Comment:
Also curious how you decided to put this here.

Reply:
Again, IntelliJ's call, but the convention is referenced [here](https://stackoverflow.com/questions/19338112/private-enum-location-inside-a-class-in-java). I checked in the settings, and we can bump enums up towards the top of the file, but figured it'd be easier to just go with their defaults.